govc/USAGE.md

```diff
@@ -4185,6 +4195,25 @@ Options:
   -t=false               Include sample times
 ```
 
+## namespace.change
+
+```
+Usage: govc namespace.change [OPTIONS]
```

Comment:
I think we should add the namespace as the positional argument?
```
Usage: govc namespace.change [OPTIONS] <namespace>
```

Reply:
Yes, I my commands weren't implementing `Usage`
I've fixed it